visualization/app/codeCharta/util/urlExtractor.ts

```diff
@@ -1,8 +1,9 @@
 "use strict"
-import { ILocationService, IHttpService } from "angular"
+import { IHttpService, ILocationService } from "angular"
 import { NameDataPair } from "../codeCharta.model"
-import { ExportCCFile, ExportWrappedCCFile } from "../codeCharta.api.model"
+import zlib from "zlib"
```

Comment:
I recommend to use another zipping library instead. This is a very old unmaintained module that was intended as a drop in replacement for the Node.js one. By now they strongly deviate and it should probably not be used anymore. I also wonder if this is automatically added by webpack as it's not listed in our package.json as dependency?